diff --git a/glob.js b/glob.js
index afcf82752c390ad569e06f23c7106bc274262252..76649ca3f0fcd93fb8d22d903227ba848d402e4a 100644
--- a/glob.js
+++ b/glob.js
@@ -42,7 +42,7 @@ module.exports = glob
 
 var rp = require('fs.realpath')
 var minimatch = require('minimatch')
-var Minimatch = minimatch.Minimatch
+// var Minimatch = minimatch.Minimatch
 var inherits = require('inherits')
 var EE = require('events').EventEmitter
 var path = require('path')
@@ -53,7 +53,7 @@ var common = require('./common.js')
 var setopts = common.setopts
 var ownProp = common.ownProp
 var inflight = require('inflight')
-var util = require('util')
+// var util = require('util')
 var childrenIgnored = common.childrenIgnored
 var isIgnored = common.isIgnored
 
diff --git a/sync.js b/sync.js
index 4f46f90559a0caad8568b44dd9d682d8e8c000f8..d4bb6803e0f16299a82ed785a6da8620d3eb1c08 100644
--- a/sync.js
+++ b/sync.js
@@ -3,9 +3,9 @@ globSync.GlobSync = GlobSync
 
 var rp = require('fs.realpath')
 var minimatch = require('minimatch')
-var Minimatch = minimatch.Minimatch
-var Glob = require('./glob.js').Glob
-var util = require('util')
+// var Minimatch = minimatch.Minimatch
+// var Glob = require('./glob.js').Glob
+// var util = require('util')
 var path = require('path')
 var assert = require('assert')
 var isAbsolute = require('path-is-absolute')